man/passwd.1.xml

```diff
@@ -94,23 +94,16 @@
       </para>
 
       <para>
-	Then, the password is tested for complexity. As a general guideline,
-	passwords should consist of 6 to 8 characters including one or more
-	characters from each of the following sets:
+	Then, the password is tested for length.  As a general guideline,
+	passwords should be long.  It's fine to use simple character sets,
+	such as passwords consisting only of lowercase letters.  For a
+	password consisting only of lowercase English letters randomly
+	chosen, and a length of 64, there are 27^64 (approximately 2^304)
```

Comment:
ok, but no one is generating & memorizing 64 randomly generated alphas.  it's one thing to use a password manager where long random generated values are never typed by users, but shadow is specifically geared towards users typing in their password interactively.

Reply:
I do remember one, for my master PGP key.  Considering I only need a handful of passwords in my life (all the rest, are pass(1)'s task), I can memorize just a few very strong ones.  The trick I use to remember them is to break them into shorter chunks (say around 10 chars each), and treat them as words in a foreign language; I can certainly remember a few dozen "words" that are originally gibberish, with some repetition; then those words are connected to make sentences which are the actual password.

However, I admit my login, my SSH and my PGP signing passwords are half of that; not because I don't remember long ones, but because typing them dozens of times an hour would be cumbersome.  But 32 random `[:lower:][:digit:]` is definitely easy, and strong enough for almost anything.